cement/ext/ext_yaml.py

```
@@ -91,13 +90,13 @@ class Meta(output.OutputHandler.Meta):
 
     def __init__(self, *args: Any, **kw: Any) -> None:
-        super(YamlOutputHandler, self).__init__(*args, **kw)
+        super().__init__(*args, **kw)
         self.config = None
 
-    def _setup(self, app_obj: App) -> None:
+    def _setup(self, app_obj: "App") -> None:
         self.app = app_obj
 
-    def render(self, data: Dict[str, Any], template: str = None, **kw: Any) -> str:  # type: ignore
+    def render(self, data: dict[str, Any], template: str = None, **kw: Any) -> str:  # type: ignore
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)